pandas/tests/plotting/test_hist_method.py

```diff
@@ -293,6 +293,28 @@ def test_hist_column_order_unchanged(self, column, expected):
 
         assert result == expected
 
+    @pytest.mark.slow
```

Comment:
is this test slow? i don't think so. do you have the ballpark of the test time for this?

Reply:
The total test takes 0.7s on my machine. Not sure what the cutoff is for slow.

Reply:
okay, 0.6s is very fast 

usually tests taking longer than at least 5s need to be marked as slow. no need to mark a <1s test